src/overview/using-the-standard-library.md

```diff
@@ -1,63 +1,33 @@
-| :------: | :--------: |
-|  ESP32   |     ✅      |
-| ESP32-C2 | _planned_  |
-| ESP32-C3 |     ✅      |
-| ESP32-S2 |     ✅      |
-| ESP32-S3 |     ✅      |
-| ESP32-H2 | _planned_  |
-| ESP8266  |     ❌      |
+[esp-rs/esp-idf-svc](https://github.com/esp-rs/esp-idf-svc) adds extra support for services/modules not available in the standard library, such as Wi-Fi management, NVS (non-volatile storage), networking services, such as `httpd` and `ping`, etc.
 
-Since `esp-idf` does not have support for the ESP8266, we unfortunately are unable to use these libraries with it. See the [Bare Metal] page for information on the `no_std` HAL for this chip.
+### Relevant `esp-rs` crates
 
-[bare metal]: ./bare-metal.md
-
-## Standard Library Features
-
-The supported `std` features are as follows:
-
-- Threads
-- Mutexes and other synchronization primitives
-- Collections
-- Random number generation
-- Sockets
-
-In addition to the `std` features, there is an [embedded-svc] implementation for `esp-idf`, [esp-idf-svc], which adds extra support for services/modules not available in the standard library, including:
-
-- Wi-Fi management
-- NVS (non-volatile storage)
-- Networking services like `httpd` and `ping`
-
-In general, this approach should feel quite similar to developing for most normal PC environments.
-
-[esp-idf]: https://github.com/espressif/esp-idf
-[newlib]: https://sourceware.org/newlib/
-[embedded-svc]: https://github.com/esp-rs/embedded-svc
-[esp-idf-svc]: https://github.com/esp-rs/esp-idf-svc
-
-## Relevant `esp-rs` crates
+[esp-idf-github]: https://github.com/espressif/esp-idf
 
 | Repository            | Description                                                                                                   |
 | --------------------- | ------------------------------------------------------------------------------------------------------------- |
-| [esp-rs/esp-idf-hal]  | An implementation of the `embedded-hal` and other traits using the `esp-idf` framework.                       |
 | [esp-rs/esp-idf-svc]  | An implementation of [embedded-svc] using `esp-idf` drivers.                                                  |
+| [esp-rs/esp-idf-hal]  | An implementation of the `embedded-hal` and other traits using the `esp-idf` framework.                       |
 | [esp-rs/esp-idf-sys]  | Rust bindings to the `esp-idf` development framework. Gives raw (`unsafe`) access to drivers, Wi-Fi and more. |
-| [esp-rs/embedded-svc] | Abstraction traits for embedded services. (`WiFi`, `Network`, `Httpd`, `Logging`, etc.)                       |
+| [esp-rs/embedded-svc] | Abstraction traits for embedded services (`WiFi`, `Network`, `Httpd`, `Logging`, etc.)                       |
+
+[newlib]: https://sourceware.org/newlib/
+[embedded-svc]: https://github.com/esp-rs/embedded-svc
+[esp-idf-svc]: https://github.com/esp-rs/esp-idf-svc
 
 The aforementioned crates have interdependencies, and this relationship can be seen below.
```

Comment:
If there are similar interdependencies between `no_std` crates, then, I believe, it will be very useful to have such a diagram. 

Can you please create one?